beacon-chain/sync/initial-sync/blocks_queue.go

```diff
@@ -376,7 +382,7 @@ func (q *blocksQueue) onProcessSkippedEvent(ctx context.Context) eventHandlerFn
 		// Only the highest epoch with skipped state can trigger extension.
 		if !m.isLast() {
 			// When a state machine stays in skipped state for too long - reset it.
-			if time.Since(m.updated) > 5*staleEpochTimeout {
+			if time.Since(m.updated) > 10*staleEpochTimeout {
```

Comment:
lets parameterize `10` , instead of just leaving it as a magic number.

Reply:
Good point, will do now.